charts/redis-ha/Chart.yaml

```diff
@@ -1,12 +1,11 @@
 apiVersion: v2
 name: redis-ha
 home: http://redis.io/
-engine: gotpl
 keywords:
 - redis
 - keyvalue
 - database
-version: 4.19.1
+version: 4.21.0
```

Comment:
Why does this version jump? What about 4.20?

Reply:
I did 4.20 in a separate PR.